sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample7_AnalyzeHealthcareEntitiesBatchConvenienceAsync.cs

```diff
@@ -34,50 +34,69 @@ public async Task HealthcareBatchAsync()
 
             string document2 = "Prescribed 100mg ibuprofen, taken twice daily.";
 
-            List<string> batchInput = new List<string>()
+            List<TextDocumentInput> batchInput = new List<TextDocumentInput>()
             {
-                document1,
-                document2,
+                new TextDocumentInput("1", document1)
+                { Language = "en" },
+                new TextDocumentInput("2", document2)
+                { Language = "en" },
+                new TextDocumentInput("3", string.Empty)
             };
+            var options = new AnalyzeHealthcareEntitiesOptions { IncludeStatistics = true };
 
-            AnalyzeHealthcareEntitiesOptions options = new AnalyzeHealthcareEntitiesOptions()
-            {
-                IncludeStatistics = true
-            };
-
-            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(batchInput, "en", options);
+            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(batchInput, options);
 
             await healthOperation.WaitForCompletionAsync();
 
+            Console.WriteLine($"Created On   : {healthOperation.CreatedOn}");
+            Console.WriteLine($"Expires On   : {healthOperation.ExpiresOn}");
+            Console.WriteLine($"Id           : {healthOperation.Id}");
```

Comment:
I wouldn't show the `Id` as this is not useful for the user. i.e. there is nothing they can do with it